### PR TITLE
Cleanup system stats

### PR DESCRIPTION
**Pull request by DiegoRBaquero**
Disk stats are now MB only and memory KB only.

### CODE REVIEW

File: container/shim/src/utils/system.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const exec = promisify(CpExec);
 
 const debug = Debug.extend("system");
 
-const meminfoKBToGB = (bytes) => bytes / 1_000_000;
+const meminfoKBToGB = (bytes) => (bytes / 1_000_000).toFixed(0);
```
Comment (guanzo) [inline container/shim/src/utils/system.js:13]: Use powers of 2 instead of 10? e.g. `1048576` (1024 ** 2) instead of `1_000_000`

Comment (DiegoRBaquero) [reply container/shim/src/utils/system.js:13]: Nope, reports incorrect info. Base 10 reports correctly for both memory and disk

Comment (DiegoRBaquero) [reply container/shim/src/utils/system.js:13]: 4027844 kB -> 4GB using base 10, 3.84 GB using base 2.

**Review by guanzo — APPROVED** [review]
lgtm

**Pull request merged**